SofaKernel/modules/Sofa.Type/CMakeLists.txt

```diff
@@ -61,14 +61,18 @@ set(SOURCE_FILES
 )
 
 sofa_find_package(Boost REQUIRED)
-
 sofa_find_package(Sofa.Config REQUIRED)
 
 add_library(${PROJECT_NAME} SHARED ${HEADER_FILES} ${SOURCE_FILES})
 
 target_link_libraries(${PROJECT_NAME} PUBLIC Sofa.Config)
 target_link_libraries(${PROJECT_NAME} PUBLIC Boost::boost)
 
+# TODO1: add an option when the sofa code has been converted
+# TODO2: if cmake >= 3.13, move those lines to SofaFramework project (or a cmakefile in module/ dir)
+add_subdirectory(../Sofa.Compat ${CMAKE_CURRENT_BINARY_DIR}/Sofa.Compat)
```

Comment:
Why here? I would put this in the main CMakeLists, right after the line `add_subdirectory(${SOFA_KERNEL_SOURCE_DIR}/modules/Sofa.Config ${CMAKE_CURRENT_BINARY_DIR}/Sofa.Config)`

Reply:
I try to avoid adding stuff in the main CmakeLists.txt but okay with your remark